package.json

```diff
@@ -9,7 +10,7 @@
     "yarn": "1.22.19"
   },
   "scripts": {
-    "postinstall": "npm run postinstall:nx && npm run postinstall:ngcc",
+    "postinstall": "npm run postinstall:nx && npm run postinstall:ngcc && git config --local core.hooksPath .githooks && chmod ug+x .githooks/*",
```

Comment:
nitpick: suggestion: lets follow the pattern seen in the previous two commands on this line and move the new code to its own script.